stm32-modules/heater-shaker/firmware/motor_task/mc_tasks.c

```diff
@@ -191,7 +191,7 @@ void MCboot( MCI_Handle_t* pMCIList[NBR_OF_MOTORS],MCT_Handle_t* pMCTList[NBR_OF
   MCT[M1].pSpeedSensorVirtual = MC_NULL;
   MCT[M1].pSpeednTorqueCtrl = pSTC[M1];
   MCT[M1].pStateMachine = &STM[M1];
-  //MCT[M1].pTemperatureSensor = (NTC_Handle_t *) pTemperatureSensor[M1];
+  MCT[M1].pTemperatureSensor = MC_NULL; //(NTC_Handle_t *) pTemperatureSensor[M1];
```

Comment:
Would we not want to eventually monitor the temperature of the motor?

Reply:
Eventually we probably do, but we don't have that thermistor right now